internal/core/src/exec/expression/Element.h

```diff
@@ -223,6 +223,16 @@ class FlatVectorElement : public MultiElement {
                     return true;
             }
         }
+        // Handle string_view -> string comparison when T is std::string
+        if constexpr (std::is_same_v<T, std::string>) {
+            if (std::holds_alternative<std::string_view>(value)) {
+                auto sv = std::get<std::string_view>(value);
+                for (const auto& v : values_) {
```

Comment:
if constexpr (std::is_same_v<T, std::string>) {
    if (auto sv = std::get_if<std::string_view>(&value)) {
        return values_.find(*sv) != values_.end();
    }
}
maybe better to understand?